action.yml

```diff
@@ -48,10 +48,14 @@ runs:
       run: |
         # Cleanup artifacts from previous run in case it crashed
         rm -rf "lychee-${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz" lychee
-        curl -sLO "https://github.com/lycheeverse/lychee/releases/download/${{ inputs.LYCHEEVERSION }}/lychee-${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz"
-        tar -xvzf "lychee-${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz"
-        rm "lychee-${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz"
-        install -t "$HOME/.local/bin" -D lychee 
+        case '${{ inputs.LYCHEEVERSION }}' in
+        v0.0*|v0.1[0-5].*) filename='lychee-${{ inputs.LYCHEEVERSION }}-x86_64-unknown-linux-gnu.tar.gz';;
+        *) filename='lychee-x86_64-unknown-linux-gnu.tar.gz'
+        esac
```

Comment:
Proposing to give a little more detail here.
I'm not sure if my comment covers all details or if it is even correct, so feel free to propose a better wording. 😅 


```suggestion

        # Backwards compatibility: Determine filename based on version
        # See https://github.com/lycheeverse/lychee-action/pull/234 for more info.
        # This workaround will be removed with the next release.
        # Please update your workflows to use the new naming scheme.
        LYCHEEVERSION="${{ inputs.LYCHEEVERSION }}"
        if [ -z "$LYCHEEVERSION" ]; then
          LYCHEEVERSION="latest"
          echo "No version specified, using latest"
        fi
        
        case "$LYCHEEVERSION" in
          v0.0*|v0.1[0-5].*)
            filename="lychee-$LYCHEEVERSION-x86_64-unknown-linux-gnu.tar.gz"
            echo "::warning::You are using an old lychee version ($LYCHEEVERSION) with a deprecated naming scheme. Please update to the latest version to ensure compatibility with future releases."
            ;;
          *)
            # Version number is no longer part of the Linux binary naming scheme.
            # Using the latest stable version instead.
            filename="lychee-x86_64-unknown-linux-gnu.tar.gz"
            ;;
        esac
        
        echo "Using filename: $filename"
```

Reply:
Turns out this isn't necessary and we don't have to notify the user. Sorry for the noise.